### PR TITLE
Deploy latest master branch to production

### PR DESCRIPTION
**Pull request by cretualexandru**
## Why
Upcoming demo next week, we need to QA the release.

## What
* [x] latest master branch is deployed to production
* [x] documentation for deploying to production is updated (if needed)
* [x] migrations are run
* [x] bootstrap script for populating the DB is ran

## Notes
Ideally, bootstrap script should not delete other existing entries.

### CODE REVIEW

**Comment by cretualexandru** [comment]
@rtud @aismail Take a look at this and let me know what you think, please! Thank you!

File: DEPLOYING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Simple procedure
+
+### 1. Check the build status of the Docker Images
+ * Go to [Docker Hub](https://hub.docker.com/r/vitaminsoftware/cami-project/) to see the build status of the project's Docker Images
+* Wait for them to be successfully built
+
+### 2. Upgrade the containers from Rancher Web UI
+* Go to [CAMI's Rancher Web UI](http://138.68.92.229:8080/) and select the **CAMI** stack. There you'll find the already set-up docker containers
+* Select the ones you know were modified and click the **Upgrade** button (up arrow icon) from the right of the entry row
```
Comment (rtud) [inline DEPLOYING.md]: @cretualexandru: aren't you supposed to first stop the containers before upgrading?

Comment (cretualexandru) [inline DEPLOYING.md]: Nope, none that I know of.

**Comment by rtud** [comment]
@cretualexandru: lgtm, if @aismail doesn't have anything to add, I think it's good to go, thank you!

**Pull request merged**